client/pages/appointmentTable.jsx

```diff
@@ -0,0 +1,201 @@
+import React from 'react';
+import AppointmentForm from './appointmentForm';
+import Table from 'react-bootstrap/Table';
+
+export default class AppointmentTable extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      userData: this.props.userData,
+      monthIndex: null,
+      selectedDate: null,
+      selectedDay: null,
+      firstDayIndex: 0,
+      firstDay: null,
+      lastDate: null,
+      openModal: false,
+      result: null,
+      table: null,
+      selectDayAttps: null
+    };
+    this.dayCallBack = this.dayCallBack.bind(this);
```

Comment:
Here you can add the `weekday` property so you can use it in multiple methods in your class:

```JAVASCRIPT
this.weekday = ['Sunday', 'Monday', 'Tuesday', 'Wednesday', 'Thursday', 'Friday', 'Saturday'];
```